src/app/interfaces/issue/IssuePredecessor.ts

```diff
@@ -0,0 +1,4 @@
+export interface IssuePredecessor {
```

Comment:
Hab in den [Code vom Backend](
https://github.com/gedankenessen/goose-backend/blob/d1ab32a1141f1f3a928d541fd403eaadf2c94969/Goose.Domain/DTOs/issues/IssueDTODetailed.cs#L22) geschaut und da wird noch das ganze Issue gespeichert. Sollte aber, wenn der Endpunkt Live ist sich genau wie hier verhalten.

@Yochyo Hast du das auf deiner TODO Liste ?